### PR TITLE
[Form] document the NumberType html5 option

### PR DESCRIPTION
**Pull request by xabbuh**
see symfony/symfony#30267

### CODE REVIEW

File: reference/forms/types/number.rst [hunk 1]
```diff
@@ -46,6 +47,14 @@ Field Options
 
 .. include:: /reference/forms/types/options/grouping.rst.inc
 
+html5
+~~~~~
+
+**type**: ``boolean`` **default**: ``false``
+
+If set to ``true``, the render HTML input will be rendered as a native HTML5
```
Comment (javiereguiluz) [inline reference/forms/types/number.rst]: `the render HTML input` -> `the HTML input` ?

File: reference/forms/types/number.rst [hunk 2]
```diff
@@ -46,6 +47,14 @@ Field Options
 
 .. include:: /reference/forms/types/options/grouping.rst.inc
 
+html5
+~~~~~
+
```
Comment (wouterj) [inline reference/forms/types/number.rst]: We're missing a `.. versionadded:: 4.3` directive

**Review by wouterj — APPROVED** [review]
The feature is merged in core, let's merge it in the docs as well

File: reference/forms/types/number.rst (hunk 2 above)
Comment (xabbuh) [reply reference/forms/types/number.rst]: `versionadded` directive added

**Comment by wouterj** [comment]
Thanks Christian.

**Pull request merged**